examples/data-objects/webflow/src/util/direction.ts

```diff
@@ -3,8 +3,6 @@
  * Licensed under the MIT License.
  */
 
-/* eslint-disable no-bitwise */
```

Comment:
Just curious: so we disabled this rule in the base config at some point? I see the bitwise operator in line 10.

Reply:
In this case, it's disabled in the package-local config. I suspect we did that in a number of places and forgot to remove the inline comments.